src/commands/live.rs

```diff
@@ -2,15 +2,47 @@ use super::{format_hours, local_time_format};
 use super::{Error, Result};
 use crate::client::{break_policy, time_entries};
 use spinner_macro::spinner_wrap;
+use std::time::Duration;
 
 #[spinner_wrap]
 pub fn status() -> Result<String> {
     Ok(match time_entries::current_entry(&super::get_session())? {
-        Some(entry) => format!("Clocked in since {}!", local_time_format(entry.start_time)),
+        Some(entry) => {
+            let mut msg = format!("Clocked in since {}", local_time_format(entry.start_time));
+
+            let regular_hours_in_seconds = (entry.regular_hours * 3600.0) as u64;
+            let regular_hours_formatted = format_seconds_as_human_readable(regular_hours_in_seconds);
+
+            let total_duration_in_seconds = (entry.duration * 3600.0) as u64;
+            let breaks_in_seconds = total_duration_in_seconds - regular_hours_in_seconds;
+            let breaks_formatted = format_seconds_as_human_readable(breaks_in_seconds);
```

Comment:
There is already a method to format hours into human readable: https://github.com/mkon/rippling-cli/blob/main/src/commands.rs#L188

Reply:
It's for formatting the time/clock and not the duration. IMO these two should be kept separate. I can move the helper there or still use the existing one if you feel strongly about it :)

Reply:
It is used for duration. It's used for example when ending the break to show the total hours of the break.

Reply:
Ach so! I missed that. Then, does it make sense to update the format to match Rippling's UI? I can do that if you agree

<img width="216" alt="Bildschirm­foto 2023-03-28 um 12 39 28" src="https://user-images.githubusercontent.com/1839554/228212184-97b811f7-d5dd-400b-b515-be6cfc386249.png">